ci/run_envoy_docker.sh

```diff
@@ -59,7 +59,7 @@ else
           && usermod -a -G pcap envoybuild \
           && chown envoybuild:envoygroup ${BUILD_DIR_MOUNT_DEST} `# Salvo unique` \
           && chown envoybuild /proc/self/fd/2 \
-          && apt-get update && apt -y install libcairo2-dev `# Salvo unique` \
+          && apt-get update && apt -y install libcairo2-dev python3-virtualenv `# Salvo unique` \
```

Comment:
Instead of adding support for on-host Python back into the image, we should work on making the Salvo build hermetic.

Here is an example PR where this was done in the Nighthawk codebase:
https://github.com/envoyproxy/nighthawk/pull/1033

Is this something you would be able to take on? We can submit the current PR as a workaround, but we should file an issue and work towards hermetic builds, as the current setup can cause unexpected version changes.

Reply:
@mum4k  Thanks for your comments! Of course, I'd like to make Salvo build hermetic, I'll create and take on the issue.

Reply:
Thank you for taking this on!